blog/meetcontentkrk-16-pazdziernik-2023/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Zapraszamy na październikowy MeetContentKRK'
+authors: tprus
+date: '2023-10-11'
+tags:
+  - 'news'
+  - 'z-kraju'
+  - 'dokumentacja'
+  - 'meetcontent'
+  - 'spolecznosc'
+coverImage: 'MeetContentKRK_pazdziernik.png'
+---
+
+Czy twoje narzędzie do dokumentacji przyprawia cię o ból głowy? Czujesz, że to dobry czas na zmiany, ale nie wiesz gdzie i jak zacząć?
```

Comment:
Wydaje się, że ten plik potrzebuje, żeby odpalić na nim Prettier. Jeśli zrobiłeś setup tak jak sugerowałem w Readme, to powinno wystarczyć, jak wciśniesz Ctrl+S.